src/Components/SmartComponents/CVEs/CVEsAssets.js

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Not that I'm opposed to having separate constant files for each page, but I think the transition to separate files should be consistent across all pages and should be changed in one go, not gradually. Some constants like `CVES_FILTER_PARAMS` and `CVES_ALLOWED_PARAMS` are still kept in `constants.js`. So now I'd have to search through 2 constant files instead of one.

Reply:
Hmm, I also support separating the effort. However, if I do not extract these constants I face a circular dependencies issue that is caused by them and the UI breaks because of reading undefined variables :)